editor/editor_interface.h

```diff
@@ -110,6 +122,14 @@ class EditorInterface : public Object {
 	String get_current_feature_profile() const;
 	void set_current_feature_profile(const String &p_profile_name);
 
+	// Editor dialogs.
+
+	// TODO: currently crashes if used when !is_editor_hint(), but https://github.com/godotengine/godot/pull/80962 should fix
+	void popup_dialog_scene_tree(const Callable &p_callback, const TypedArray<StringName> &p_valid_types = TypedArray<StringName>());
+	// Must use Vector<int> because exposing Vector<Variant::Type> is not supported.
+	void popup_dialog_property_selector(Object *p_object, const Callable &p_callback, const Vector<int> &p_type_filter = Vector<int>());
+	// TODO: Should there also be methods for PropertySelector::select_property_from_base_type, select_property_from_script, and select_property_from_basic_type?
```

Comment:
We don't usually use TODOs to ask ourselves questions. TODOs are for things which should be done. This can be a question in the PR itself.